controllers/eventbus/validate.go

```diff
@@ -32,9 +32,6 @@ func ValidateEventBus(eb *v1alpha1.EventBus) error {
 		if x.Version == "" {
 			return fmt.Errorf("invalid spec: a version for jetstream needs to be specified")
 		}
-		if x.Replicas != nil && *x.Replicas < 3 {
```

Comment:
To disallow `replicas==2` or `replicas<=0`?

Reply:
Yes good point, I'll add that check now